internal/host/conditions.go

```diff
@@ -29,36 +29,60 @@ func (c conditionId) String() string {
 }
 
 func (v *validator) isInstallationDiskSpeedCheckSuccessful(c *validationContext) bool {
+	if c.infraEnv != nil {
```

Comment:
Not sure all these changes are needed because they are checked in certain states.  For example here https://github.com/openshift/assisted-service/blob/fa49588aa60a275a2e9d6bb4f11ab9320e99f40c/internal/host/statemachine.go#L314

Reply:
since it is preprocessed, i want to make sure that cluster is not access in case of infra-env host. In addition, you will probably change this code when dealing with MGMT-7366 :)